tests/system_tests/external_interaction/test_ispyb_dev_connection.py

```diff
@@ -845,7 +845,7 @@ def test_ispyb_deposition_in_rotation_plan(
     position_id = fetch_datacollection_attribute(
         dcid, DATA_COLLECTION_COLUMN_MAP["positionid"]
     )
-    expected_values = {"posX": 10.0, "posY": 20.0, "posZ": 30.0}
+    expected_values = {"posX": 1.0, "posY": 2.0, "posZ": 3.0}
```

Comment:
since we are now getting the info from the hardware, the expected values now come from the smargon fixture which had those xyz values which are coincidentally a factor of 10x smaller